flink-ml-parent/flink-ml-lib/src/main/java/org/apache/flink/ml/common/dataproc/vector/VectorToColumnsMapper.java

```diff
@@ -0,0 +1,104 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package org.apache.flink.ml.common.dataproc.vector;
```

Comment:
I was wondering if this is the right package to put this utility function in, maybe:
`org.apache.flink.ml.common.utils.dataproc` makes more sense?

This is especially confusing since we already have `org.apache.flink.ml.common.mapper` base class.

Reply:
This is not a utility class, it's a data preprocess mapper. So I think it shouldn't be in a util package.

Reply:
I see. I am assuming this pre-processing is similar to, say feature normalization, or some specific pre-processing algorithm such as: [Word Embedding algorithm](https://en.wikipedia.org/wiki/Word_embedding)?

Reply:
yes, but this is a pretty simple one.

Reply:
in this case I think the location is spot on :-) Thanks